face_it/settings/base.py

```diff
@@ -92,45 +91,10 @@
     'django.contrib.staticfiles.finders.AppDirectoriesFinder',
     #    'django.contrib.staticfiles.finders.DefaultStorageFinder',
     'compressor.finders.CompressorFinder',
-    'pipeline.finders.PipelineFinder',
 )
 
 #Use Django Pipeline for static file storage
-STATICFILES_STORAGE = 'pipeline.storage.PipelineCachedStorage'
-
-#No Compressing for now TODO
-PIPELINE_CSS_COMPRESSOR = 'pipeline.compressors.NoopCompressor'
-PIPELINE_JS_COMPRESSOR = 'pipeline.compressors.NoopCompressor'
-
-# CSS Files. Eventually remove CDN TODO
-PIPELINE_CSS = {
-    # Project libraries.
-    'css': {
-        'source_filenames': (
-            'bower_components/bootstrap-social/bootstrap-social.css',
-            'bower_components/font-awesome/css/font-awesome.css',
-            'css/*.css',
-        ),
-        # Compress passed libraries and have
-        # the output in`css/css.min.css`.
-        'output_filename': 'css/css.min.css',
-        'variant': 'datauri',
-    }
-    # ...
-}
-# JavaScript files.
-PIPELINE_JS = {
-    # Project JavaScript libraries.
-    'js': {
-        'source_filenames': (
-            'bower_components/underscore/underscore.js',
-            'js/compress/*.js',
-        ),
-        # Compress all passed files into `js/js.min.js`.
-        'output_filename': 'js/js.min.js',
-    }
-    # ...
-}
+STATICFILES_STORAGE = 'whitenoise.django.GzipManifestStaticFilesStorage'
```

Comment:
I understand that whitenoise was added to make life easier for heroku.  A couple questions:
- Are there any issues with this setup in the vagrant box
- I'm curious if the whitenoise static files are properly minified, and if not could that have an impact on performance?